projects/Kbin/default.nix

```diff
@@ -5,7 +5,10 @@
 } @ args: {
   packages = {inherit (pkgs) kbin kbin-frontend kbin-backend;};
   nixos = {
-    modules.service = ./service.nix;
+    modules.service = {
+      path = ./service.nix;
+      options = "services\\.kbin\\..*";
+    };
```

Comment:
That proposal works out for NixOS modules, yes. But not for, say, home-manager. We could do that by inverting things...

```nix
{
  modules = {
    nixos.services.kbin = ./service.nix;
    homeManager.programs.kbin-client-tool = ./hm.nix;
  };
}
```

But that's also not nice. Because we already have `nixos.{tests,configurations}`. So I suggest that we would add `homeManager` at the NixOS level once we actually have non-NixOS modules:

```
{
  nixos.modules.services.kbin = ./services.nix;
  homeManager.modules.programs.kbin-client-tool = ./hm.nix;
}
```

Reply:
Wait... We also have to flatten the module names for flake outputs. So these names would get longer, e.g. `nixosModules."KBin/services/kbin"` but we could use the option path itself only, e.g. `nixosModules."services.kbin"`.

Reply:
>  I suggest that we would add homeManager at the NixOS level once we actually have non-NixOS modules

That's what I had in mind, yes.